packages/@featherds/styles/themes/_base-light.scss

```diff
@@ -14,19 +14,57 @@ $colors: (
   $state-color-on-color: #fff,
   $state-color-on-neutral: #000,
   //added here so RGB is added
-  $primary-text-on-surface: rgba(10, 12, 27, 0.9)
+  $primary-text-on-surface: rgba(10, 12, 27, 0.9),
 );
 
 $colors: add-RGB($colors);
 
 $hero-gradients: (
-  $hero-gradient-1-1: radial-gradient(ellipse 200% 500% at -90% 0%, rgba(#DB2973, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-1-2: radial-gradient(ellipse 100% 250% at 130% 180%, rgba(#32E7F3, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-1-3: radial-gradient(ellipse 0% 0% at 0% 0%, transparent 0%, transparent 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-2-1: radial-gradient(ellipse 200% 200% at -100% -110%, rgba(#DB2973, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-2-2: radial-gradient(ellipse 100% 250% at 130% -115%, rgba(#32E7F3, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-2-3: radial-gradient(ellipse 100% 250% at 110% 250%, rgba(#3A4BD3, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
-  $hero-gradient-2-4: radial-gradient(ellipse 200% 200% at -90% 160%, rgba(#3D50C5, .5) 0%, #14D1DF00 100%) 0% 0% no-repeat padding-box,
+  $hero-gradient-1-1:
+    radial-gradient(
+      ellipse 200% 500% at -90% 0%,
+      rgba(#db2973, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
+  $hero-gradient-1-2:
+    radial-gradient(
+      ellipse 100% 250% at 130% 180%,
+      rgba(#32e7f3, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
+  $hero-gradient-1-3:
+    radial-gradient(ellipse 0% 0% at 0% 0%, transparent 0%, transparent 100%) 0%
+    0% no-repeat padding-box,
+  $hero-gradient-2-1:
+    radial-gradient(
+      ellipse 200% 200% at -100% -110%,
+      rgba(#db2973, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
+  $hero-gradient-2-2:
+    radial-gradient(
+      ellipse 100% 250% at 130% -115%,
+      rgba(#32e7f3, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
+  $hero-gradient-2-3:
+    radial-gradient(
+      ellipse 100% 250% at 110% 250%,
+      rgba(#3a4bd3, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
+  $hero-gradient-2-4:
+    radial-gradient(
+      ellipse 200% 200% at -90% 160%,
+      rgba(#3d50c5, 0.5) 0%,
+      #14d1df00 100%
+    )
+    0% 0% no-repeat padding-box,
 );
 
 $on-color: (
```

Comment:
don't think we should change these values to rem

Reply:
Resolved